docs/src/pkgs/index.md

```diff
@@ -0,0 +1,70 @@
+# [Packages under JuliaImages](@id page_packages_index)
+
+!!! tip
+    It's recommended to gone through the tutorials before you dive into the package documentation to get
+    a basic understanding of the JuliaImages ecosystem. Otherwise, you might find yourself lost in the
+    details and get frustrated.
+
+🚧 This section consists of documentations of packages that are closely related to JuliaImages ecosystem.
+Most of them are also maintained by members of JuliaImages. They're ordered alphabetically so you can
```

Comment:
```suggestion
The ones marked with a `*` are available via `using Images`, but you can also use packages individually. Below, they're grouped into broad categories, then ordered alphabetically so you can
```
I wondered if explaining the star at the top would be more helpful than at the bottom, but I can go either way, your choice.

Reply:
These provide links to the packages themselves, which seems useful, but should we say something about aggregating the documentation here?